sycl/doc/extensions/README.md

```diff
@@ -7,8 +7,6 @@ DPC++ extensions status:
 
 |  Extension  |    Status   |   Comment   |
 |-------------|:------------|:------------|
-| [SPV_INTEL_function_pointers](SPIRV/SPV_INTEL_function_pointers.asciidoc)                                                   | Supported(OpenCL: CPU, GPU; HOST)         | |
-| [SPV_INTEL_inline_assembly](SPIRV/SPV_INTEL_inline_assembly.asciidoc)                                                       | Supported(OpenCL: GPU)                    | |
 
```

Comment:
It it the right time to remove this file?

Reply:
I am preparing another commit with replaces this file, and also adds some more README files.  If you want this file removed sooner, I can remove it in this PR.  Do you want that, or is it OK to wait for my next PR?

Reply:
Next PR sounds good to me.